config/open-feature/org.yaml

```
@@ -34,6 +35,7 @@ members:
   - agentgonzo
   - ajhelsby
   - alina-v1
+  - AlexJones
```

```suggestion
  - AlexsJones
```

you do have a typo in this user name

---

Thank you, thought I had messed other stuff up